### PR TITLE
update(OWNERS): move inactive maintainers to emeritus approvers

### PR DESCRIPTION
**Pull request by jasondellaluce**
**What type of PR is this?**

/kind documentation

**Any specific area of the project related to this PR?**

**What this PR does / why we need it**:

For https://github.com/falcosecurity/evolution/issues/157 (see: https://github.com/falcosecurity/evolution/issues/157), this PR proposes to move to `emeritus_approvers` maintainers that have been inactive in the past 6 months, as for our governance (see: https://github.com/falcosecurity/.github/blob/master/GOVERNANCE.md#project-inactivity). Maintainers are invited to have a discussion and take a decision.

cc @falcosecurity/maintainers 

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

@leodido has non-zero devstats in the past 6 months, however from https://github.com/falcosecurity/falco/issues/2106#issuecomment-1182608352 I understand he intends to step down. If not, please have a discussion below.

### CODE REVIEW

**Review by leogr — COMMENTED** [review]
This particular repository does not require much involvement but needs attention. So I'm ok with moving those owners to emeritus approvers because they egregiously served the Falco project in the past, but now it seems they don't have time anymore.

However, I don't feel comfortable remaining the only approver on the list. For this reason, I'd like to propose @maxgio92 and @cpanato, who have historically been involved in discussions regarding the whole organization. IMO they have all the requirements to serve this repository. I'd also ask some maintainers from @falcosecurity/falco-maintainers to help here (@jasondellaluce are you interested?).

Please let us know.

Thanks

**Comment by leodido** [comment]
@jasondellaluce I think it makes no sense at all to look at `devstats` for the "special" repositories.
Especially, when we're talking about the governance repository (this one). Changes to such a repository are vital to the healthy setup of an OSS organization...

Let me state once again that also this seems a unilateral decision that has not been discussed with all the **current** maintainers.

Anyways, since **I do not agree in any way with how you all are doing all this**, even if I've been trying to keep this healthy in the past months, **my moral compass impedes me from sharing any kind of association or commitment with you all here**.

Even tho I stepped down as a Falco maintainer, it doesn't seem that leaving only a single person here and that person proposing their own people is a sign of a healthy OSS organization.

Again, **I disassociate myself from anything happening here**.

Maybe @caniszczyk should take a look at this?

**Comment by leogr** [comment]
> Even tho I stepped down as a Falco maintainer, it doesn't seem that leaving only a single person here and that person proposing their own people is a sign of a healthy OSS organization.

Hey @leodido 

As I have tried to repeat several times, this PR is not a taken decision. This PR is a proposal and an invite to discuss. At this point, I really believe you misunderstood our intentions. 

Switching back to the main discussion. I agree with you that having a single person here is not ok (see my [comment](https://github.com/falcosecurity/.github/pull/84#pullrequestreview-1038652842) above). Indeed, I'm trying to find new people to help here. 

By the way, I understood you stepped down, but I want to let you know that I'm fine to have you here if you want to remain. This is a special repository, and it does not require much effort. Please, consider that.

**Comment by maxgio92** [comment]
I'd like to help here. But please judge you maintainers, if I'm valid for the duty of a maintainer of such important repository.

I agree also that activity monitoring of a maintainer of this type of repository can't follow same development monitoring that is done for code repositories. I'm not sure a right metric exists for that.

**Comment by jasondellaluce** [comment]
> @jasondellaluce I think it makes no sense at all to look at devstats for the "special" repositories.
Especially, when we're talking about the governance repository (this one). Changes to such a repository are vital to the healthy setup of an OSS organization...

@leodido I agree with you, but at the same time I don't see what the right metric would be to estimate activity in special repositories. Do you have better proposals? I would also add that most maintainers of these repositories have been added to OWNERS before our governance was in place, so I think it would be unfair towards the community to avoid reviewing the list. Again, I would highlight that I opened this PR as a proposal: the goal here is to allow everyone to have this specific kind of conversation.

**Comment by leogr** [comment]
I've added the `wip:` prefix (ie. work in progress) to this PR to make clear that the current proposed change is not the definitive decision.

File: OWNERS
Comment (leogr) [inline OWNERS:4]:
```suggestion
  - maxgio92
emeritus_approvers:
```

Since @maxgio92 diligently served the Falco project and maintained the https://github.com/falcosecurity/evolution repository, I believe having him on board here will be beneficial.
This is my evaluation criteria.

**Review by leogr — COMMENTED** [review]
As per my [comment above](https://github.com/falcosecurity/.github/pull/84#pullrequestreview-1038652842), I believe we need more than one maintainer here.

Since @maxgio92 accepted, I think we are set now.

I want just to remind anyone, that if someone else is still willing to help, they can propose themself at any time.

**Review by leogr — APPROVED** [review]
The maintainers moved to emeritus:
- have been inactive for the past 6 months,
- have stepped down,
- or are unreachable.

Now, since at least two maintainers (from different companies) are now willing to take care of this repository, I think this change can be merged.

I want to remind you again that anyone willing to help here can propose themself. We will be happy to discuss it together.

**Pull request merged**
